packages/atomic/src/components/facets/atomic-rating-facet/atomic-rating-facet.tsx

```diff
@@ -109,6 +109,13 @@ export class AtomicRatingFacet
    * - Use a value that starts with `http://`, `https://`, `./`, or `../`, to fetch and display an icon from a given location.
    * - Use a value that starts with `assets://`, to display an icon from the Atomic package.
    * - Use a stringified SVG to display it directly.
+   *
+   * When using a custom icon, at least part of your icon should have the color set to `fill="currentColor"`.
+   * This part of the SVG will take on the colors set in the following variables:
```

Comment:
I would maybe make variables clickable with this link https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties